Muszilla/src/Controller/HomeController.php

```diff
@@ -0,0 +1,309 @@
+<?php
+
+namespace App\Controller;
+
+use App\Controller\registerController;
+use App\model\DbConnect;
+use App\model\User;
+use App\model\Music;
+use Exception;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * Class to handle all the routes available in the Music web app. It handle the 
+ * routing and the operations involved with that particular route.
+ * 
+ * @var object $conn
+ *    Variable to store the mysqli database connection object.
+ * 
+ * @var object $music 
+ *    Variable to store the object of the Music class.
+ * 
+ * @var object $connection_obj 
+ *    Variable to store the object od the DbConnect class.
+ * 
+ * @var object $user 
+ *    Variable to store the object of the User class.
+ * 
+ * @var object $upload_obj
+ *    Variable to store the object of the uploadCon class.
+ */ 
+class HomeController extends AbstractController {
+
+  private object $conn = '';
```

Comment:
Comment about these varaibles.

Reply:
Already added in class description. Kindly check.